src/core/thread/mesh_forwarder.cpp

```diff
@@ -672,6 +669,11 @@ ThreadError MeshForwarder::UpdateIp6Route(Message &aMessage)
 
         break;
 
+#if OPENTHREAD_FTD
+        uint16_t rloc16;
```

Comment:
style: prefer to wrap the Router/Leader case block in `{` and `}` and declare this local variables within that code block.